jinja2_standalone_compiler/__init__.py

```diff
@@ -110,34 +116,47 @@ def main(path, settings=None, out_path=None, verbose=False, silent=False):
             os.unlink(template_file)
             raise
 
-    if not silent:
-        print 'Done!'
-
 
 @click.command()
 @click.argument('path', type=click.Path(exists=True))
-@click.option('--settings', default=None, help='Settings file to use.')
-@click.option('--out', type=click.Path(exists=False, file_okay=False, dir_okay=True), default=None, help='Output path to use.')
+@click.option('--settings', '-s', default=None, multiple=True, help='Settings file to use.')
+@click.option('--out', '-o', type=click.Path(exists=False, file_okay=False, dir_okay=True), default=None, help='Output path to use.')
 @click.option('--verbose', is_flag=True, default=False, help='Detailed command line output')
 @click.option('--silent', is_flag=True, default=False, help='Suppress command line output')
 def main_command(path, settings=None, out=None, verbose=False, silent=False):
+    init(autoreset=True)  # Init colorama
     current_dir = os.getcwd()
 
-    if settings:
-        settings_file = os.path.join(current_dir, settings)
-        if not os.path.exists(settings_file):
-            raise IOError(u'Settings file not found: {}'.format(settings_file))
-
-        sys.path.insert(0, '')
-        settings = imp.load_source(current_dir, settings)
-
-    current_dir = os.path.join(current_dir, path)
+    global_silent = silent
```

Comment:
Shouldn't you reference those vars  with `global global_silent/global_verbose` before using them (in order to let pyhon knows you're changing a global var instead of just creating a local var)?